app.rb

```diff
@@ -0,0 +1,150 @@
+require 'sinatra/base'
+require 'sinatra/reloader'
+require_relative 'lib/peep_repository'
+require_relative 'lib/user_repository'
+require_relative 'lib/database_connection'
+
+DatabaseConnection.connect
+
+class Application < Sinatra::Base
+  # This allows the app code to refresh
+  # without having to restart the server.
+  configure :development do
+    register Sinatra::Reloader
+    also_reload 'lib/peep_repository'
+    also_reload 'lib/user_repository'
+  end
+
+  enable :sessions
```

Comment:
Interestingly when I left this `enable :sessions` outside of the `configure` block, it wouldn't work for me